### PR TITLE
update to v3

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

Tests failing because fastify-cookie needs to be update to v3 as well

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,14 @@
   "version": "1.3.1",
   "description": "JWT utils for Fastify",
   "main": "jwt.js",
-  "types": "index.d.ts",
+  "types": "jwt.d.ts",
   "engines": {
-    "node": ">6"
+    "node": ">10"
```
Comment (Eomm) [inline package.json:8]:
```suggestion
    "node": ">=10"
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        node: ['14', '12', '10']
+      name: Node ${{ matrix.node }}
+      steps:
+        - uses: actions/checkout@v1
+        - name: Setup node
+          uses: actions/setup-node@v1
+          with:
+            node-version: ${{ matrix.node }}
+        - run: npm install
+        - run: npm run standard
```
Comment (Eomm) [inline .github/workflows/ci.yml:18]: `standard` is not in the `scripts` (it is included in `test`)
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['14', '12', '10']
+      name: Node ${{ matrix.node }}
+      steps:
+        - uses: actions/checkout@v1
```
Comment (Eomm) [inline .github/workflows/ci.yml:12]:
```suggestion
        - uses: actions/checkout@v2
```

**Comment by Ethan-Arrowood** [comment]
CI will fail until fastify-cookie is updated. @mcollina am I okay to merge or should I get a v3 branch of cookie first and then merge?

**Comment by mcollina** [comment]
> CI will fail until fastify-cookie is updated. @mcollina am I okay to merge or should I get a v3 branch of cookie first and then merge?

You can point to https://github.com/fastify/fastify-cookie/pull/67. Or I can just land and ship that.

**Comment by Ethan-Arrowood** [comment]
I'm going to land because this isn't dependent on cookie; thats just an implementation test. I'll open an issue to update when cookie is released

**Pull request merged**